tests/Feature/Importing/Api/ImportAssetsTest.php

```diff
@@ -109,6 +112,14 @@ public function importAsset(): void
         $this->assertEquals($row['purchaseDate'], $newAsset->purchase_date->toDateString());
         $this->assertNull($newAsset->asset_eol_date);
         $this->assertEquals(0, $newAsset->eol_explicit);
+        /*
+         * FIXME FIXME FIXME please - I don't know what the right thing to do here is.
+        //Hrm. I'm not sure about this. So it's saying that the asset's location should be the same as the asset's rtd_location_id - but that doesn't make sense to me.
+        If an asset is assigned to a *user* who has no location, the *asset* should get no location. The user definitely has no location - I've dumped them, I've looked
+        at the factories. So the asset's location should be changed from its rtd_location_id to the User's location_id - which is definitively null. So I think this test
+        might be wrong? But, at the same time, if that's something we've been doing for years, we might have to keep doing that for years. Which is weird. Very, very weird.
+        Not sure what to do on this one.
+        */
```

Comment:
I think this might have already handled that? https://github.com/grokability/snipe-it/pull/16682